### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
BTW I get 4 failed tests in script and style minification.

```
Mozilla/5.0 (Windows NT 6.1; rv:29.0) Gecko/20100101 Firefox/29.0

Tests completed in 106 milliseconds.
366 assertions of 370 passed, 4 failed.
```

### CODE REVIEW

File: src/htmlminifier.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* global CSSOCompressor: true, CSSOTranslator: true, cleanInfo: true, srcToCSSP: true */
```
Comment (kangax) [inline src/htmlminifier.js:1]: Hm... yeah, it does make sense to localize these. Thanks.

**Comment by kangax** [comment]
> 366 assertions of 370 passed, 4 failed.

Ahha... I referenced minifier files on a test page and it's (almost) good now. We should really package js/css minifiers together with minifier. We'd probably need to make it modular so that you could specify whether you want them included in a build.

**Comment by XhmikosR** [comment]
Now only one test fails :)

**Comment by kangax** [comment]
Yep, it's the difference in "o" vs. "n" for the mangled variable : )

**Comment by XhmikosR** [comment]
Maybe disable mangle in uglify for the tests?

**Comment by kangax** [comment]
We could, but I wanted to actually ensure that mangling is happening as that's one of the main things that reduces string size.

**Comment by XhmikosR** [comment]
I suggested to disable it for the tests only if possible. Unless you have another workaround on your mind of course :)

**Pull request merged**